addons/crm/wizard/crm_lead_pls_update.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class CrmUpdateProbabilities(models.TransientModel):
+    _name = 'crm.lead.pls.update'
+    _description = "Update the probabilities"
+
+    def _get_default_pls_start_date(self):
+        pls_start_date_config = self.env['ir.config_parameter'].sudo().get_param('crm.pls_start_date')
+        return fields.Date.to_date(pls_start_date_config)
```

Comment:
Is False (or None or '') working ?

Reply:
If so ( for '' values), you won't be able to open setting page itself as it's not handled in the existing compute method. It should be handled separately as a bug-fix in stable, after which there should't be such case here.

Reply:
Maybe you could make a separate commit that avoid any crash if someone removed crm.pls_start_date parameter. Then here same, at least support a void parameter even if nothing is done (raise something, display something, ...). That way we avoid unnecessary crashes.